Formula/pmdmini.rb

```diff
@@ -29,7 +29,7 @@ def install
 
     # Makefile doesn't build a dylib
     system "#{ENV.cc} -dynamiclib -install_name #{lib}/libpmdmini.dylib " \
-                     "-o libpmdmini.dylib -undefined dynamic_lookup obj/*.o"
+           "-o libpmdmini.dylib -undefined dynamic_lookup obj/*.o"
```

Comment:
The right fix here is to not pass one long string to `system`. We can use `*Dir["obj/*.o"]` to do the globbing.